### ISSUE TITLE
Edit name username 

### ISSUE THREAD

**Issue by cmos486** [louislam/uptime-kuma#1243]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

How can I change the initial username?

### 🐻 Uptime-Kuma Version

1.11.3

### 💻 Operating System and Arch

Ubuntu 20.04

### 🌐 Browser

Firefox 96

**Comment by lmigo** [comment]
Hi, how can I change the initial username?

**Comment by HigH-HawK** [comment]
Hey, we have come across this as well. We setup an instance with a name and a password without realising that we would be unable to change the username later.

Is there any way one can change the initially setup username?

I noticed a DB file (`kuma.db`) which I assume is a SQLite database file. I tried opening this file with a SQLite Browser application but it is asking to provide a password that has been used to encrypt the database. I am however not sure where to find that password.

It would be great to get an ability to change the initially setup username.
